scripts/windows-setup.ps1

```diff
@@ -48,13 +48,15 @@ if (!$SkipTools) {
 
         Invoke-WebRequest -UseBasicParsing -Uri 'https://get.scoop.sh' `
             | Invoke-Expression
+        scoop install git go nvm
         scoop bucket add versions
-        scoop install git go nvm python27
+        scoop install python
+        # Install and use latest node 16* version
+        nvm install 16
```

Comment:
@mook-as raised a good point earlier, `nvm install 16` will install `16.0.0` when using newer versions of `nvm-windows`. It's probably best to explicitly specify the version here, unless there's a better approach that I'm not aware of.

We most recently tested on `16.13.1` while working on #1198.

Reply:
On my windows machine, I have nvm 1.1.9 which seems to be the latest nvm version. When I do `nvm install 16` it pulls the latest 16 version which is 16.13.2. Unless we want to restrict to the version (16.13.1) we tested, the provided instructions should work.

Reply:
I think it's good as long as it will install the latest version of node 16 instead of 16.0.0.

Reply:
Right, turns out the problem was on my side (I had nvm-windows 1.1.7); upgraded that and things work fine now.